README.md

```diff
@@ -164,6 +164,23 @@ in the PKCS#11 token, but if that is not sufficient to uniquely identify
 the certificate, further refinement of matching certificates can be
 achieved through the `--cert-selector` flag.
 
+#### TPMv2 Integration
```

Comment:
Minor, but it might be good to include some documentation around `TPM_DEVICE` here. Or if there's a resource we can link to, maybe that would be better.